R/EXADBI-driver.R

```diff
@@ -0,0 +1,46 @@
+#' @include EXADBI-object.R
+NULL
+
+## EXADriver declaration and definition of driver instation functions.
+
+#' An interface driver object to the EXASOL Database.
+#'
+#' @seealso \code{\link{DBIDriver-class}}
+#' @family DBI classes
+#' @family EXADriver related objects
+#' @slot odbc_drv A string containing the path to the EXASOL ODBC driver file.
+#' @export
+#' @author EXASOL AG <opensource@exasol.com>
+setClass("EXADriver",
+         contains = c("DBIDriver", "EXAObject"),
+         slots = c(odbc_drv = "character")
+)
+
+#' Instantiates an EXADriver object.
+#' @family EXADriver related objects
+#' @param driver The path to an ODBC driver file. If missing, the driver installed on the system is used.
+#' If "SYSTEM": the EXASOL ODBC driver installed on the system is used immediately.
+#' Alternatively a path to an ODBC driver library can be provided.
+#' @param silent If TRUE, no message is print.
+#' @return An EXADriver object.
+#' @author EXASOL AG <opensource@exasol.com>
+#' @export
+exasol_driver <- function(driver = NULL, silent = FALSE) {
+
+  #TODO: determine driver file according to OS
+  # path.package("exasol")
```

Comment:
As above let's remove these or create an additional issue if not resolved yet.

Reply:
Maybe we could discuss this offline